internal/controller/apiextensions/composite/reconciler.go

```diff
@@ -658,3 +670,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	xr.SetConditions(xpv1.Available())
 	return reconcile.Result{RequeueAfter: r.pollInterval}, errors.Wrap(r.client.Status().Update(ctx, xr), errUpdateStatus)
 }
+
+func firstNAndSomeMore(names []string) string {
```

Comment:
given that https://github.com/crossplane/crossplane-runtime/pull/538 got merged, would you like to use the new helper function?

Reply:
done